build_tools/ci/run_matmul_test.sh

```diff
@@ -341,6 +341,18 @@ run_matmul_test \
     --vitis_path  "${VITIS}" \
     --pipeline "simple-pack"
 
+run_matmul_test \
+    --name "matmul_bf16_bf16_large_amd-aie_xrt_simple-pack" \
+    --lhs_rhs_type "bf16" \
+    --acc_type "f32" \
+    --shapes "large" \
```

Comment:
So for the large shape, currently we have one "accumulate=false" type matmul. This is the zero-fill matmul. But from the lit test and your pass you are currently only supporting accumulate=true matmul due to the air bug. Can you please clarify?

Reply:
Fill is still supported, just not vectorized. It's when we run `linalg::vectorize(rewriter, fillOp)` that the bug is hit.  accumulate=true is supported.

Reply:
Thanks, makes sense.